pytorch_pfn_extras/utils/comparer.py

```diff
@@ -105,6 +106,9 @@ def __init__(
                 A set of keys of output dict to compare.
             compare_fn (function):
                 Comparison function. Default is ``get_default_comparer()``.
+            concurrency (int, optional):
+                The upper bound limit on the number of workers that run concurrently.
```

Comment:
```suggestion
                Maximum number of workers that run concurrently.
```

Maybe this is a bit clearer? just a nit.

Reply:
Fixed. Thanks!